src/EFCore.SingleStore/Infrastructure/SingleStoreServerVersion.cs

```diff
@@ -90,6 +90,17 @@ internal SingleStoreServerVersionSupport([NotNull] ServerVersion serverVersion)
             public override bool ConnectionAttributes => ServerVersion.Version >= new Version(8, 1, 0);
             public override bool DescendingIndexes => false;
             public override bool CommonTableExpressions => false;
+            public override bool LimitWithinInAllAnySomeSubquery => false;
```

Comment:
SingleStore may differ from MySQL in these capabilities, please create a task (and refer it here) to check them later.